.github/pull_request_template.md

```diff
@@ -4,7 +4,9 @@ This code review checklist is intended to serve as a starting point for the auth
 <!-- NOTE: Do not modify these when initially opening the pull request.  This is a checklist template that you tick off AFTER the pull request is created. -->
 - [ ] Self-review: Did you review your own code in GitHub's web interface? _Code often looks different when reviewing the diff in a browser, making it easier to spot potential bugs._
 - [ ] Automated tests: Did you add appropriate automated tests for any code changes?
-- [ ] Manual tests: Did you update the [manual tests](../blob/main/docs/tests) as appropriate? _While we aim for automated testing of the application, some aspects require manual testing. If you had to manually test something during development of this pull request, write those steps down._
+- [ ] Manual tests: Did you update the [manual tests](../blob/main/docs/testing/manual_testing) as appropriate? 
+  _While we aim 
```

Comment:
This newline looks like a typo.  If you use Android Studio as a text editor, I think it will insert these.

Reply:
Yes, I also noticed, thanks. It was IDEA, which I use to write documentation.